samcli/lib/build/app_builder.py

```diff
@@ -253,8 +264,8 @@ def _build_lambda_image(self, function_name, metadata):
 
         LOG.info("Building image for %s function", function_name)
 
-        dockerfile = metadata.get("Dockerfile")
-        docker_context = metadata.get("DockerContext")
+        dockerfile = cast(str, metadata.get("Dockerfile"))
```

Comment:
rename to `docker_file` for consistency?

Reply:
I think `docker_context` was named `docker_context` instead of `dockercontext` because the `C` was uppercase, which does not apply to `Dockerfile`, which seems to be a whole word.